scripts/check-coverage.jsh

```diff
@@ -97,8 +97,8 @@ var moduleCoverageData = List.of(
   new ModuleCoverage("sirius-components-task-edit", 3.0),
   new ModuleCoverage("sirius-components-task-starter", 73.0),
   new ModuleCoverage("sirius-components-flow-starter", 79.0),
-  new ModuleCoverage("sirius-components-view-diagram-customnodes", 34.0),
-  new ModuleCoverage("sirius-components-view-diagram-customnodes-edit", 11.0),
+  new ModuleCoverage("sirius-components-view-diagram-customnodes", 37.0),
+  new ModuleCoverage("sirius-components-view-diagram-customnodes-edit", 10.0),
```

Comment:
Why should the code coverage of this part be reduced?

Reply:
I didn't know how to increase the coverage on the `view-diagram-customnodes-edit` module.

Reply:
It's fairly easy, you just need to add a test which will leverage some edit capabilities, for example something that will:

- display the label of an `EllipseNodeStyleDescription`
- request the list of its properties
- edit on of its properties

The best way to do it would be to try to interact with the details view of an `EllipseNodeStyleDescription`. Display the details and update a value like the border size. For that you could add a test in a class `org.eclipse.sirius.web.application.controllers.forms.CustomNodeDetailsViewControllerTests` which would use `DetailsViewControllerIntegrationTests` as an inspiration to subscribe to the details view of an `EllipseNodeStyleDescription`.

Since we don't already have a view based diagram description using ellipse as style, it will be the perfect opportunity to add a new one. You can just add a new view document in `studio.sql` just like `Diagram View` named for example `Ellipse Diagram View` which would use this style to display the humans. After that, you can just subscribe to the details view of this style and try to edit its border size (see TextfieldControllerTests to see how to edit a textfield for example).

Reply:
Test added